src/webserver/http.rs

```diff
@@ -600,3 +606,25 @@ fn bind_error(e: std::io::Error, listen_on: std::net::SocketAddr) -> anyhow::Err
     };
     anyhow::anyhow!(e).context(ctx)
 }
+
+fn bind_uds_error(e: std::io::Error, unix_socket: &PathBuf) -> anyhow::Error {
+    let ctx = match e.kind() {
+        std::io::ErrorKind::AddrInUse => format!(
```

Comment:
Can this error be returned when binding to a socket?

Reply:
Could you test the various error cases locally to check what ioerrors are actually returned?